### ISSUE TITLE
Truncated Energy - Test Coverage or Code Quality Don't Exist (Trac #1221)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1215]
There are no folders for:

* Test Coverage: resources/test
* Code Quality: private/test

Please add both.

Migrated from https://code.icecube.wisc.edu/ticket/1221
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:57", 
    "description": "There are no folders for:\n\n* Test Coverage: resources/test\n* Code Quality: private/test\n\nPlease add both.", 
    "reporter": "jtatar", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067117911749", 
    "component": "combo reconstruction", 
    "summary": "Truncated Energy - Test Coverage or Code Quality Don't Exist", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2015-08-19T20:21:25", 
    "milestone": "", 
    "owner": "jtatar", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20150826T02:16:19: 
* olivas changed owner from "smiarecki" to "jtatar"
* olivas changed status from "new" to "assigned"

**Comment by icecube-trac** [comment]
Trac update at 20151029T11:36:51: 
* olivas changed priority from "normal" to "blocker"
* olivas commented:

> At the very least we need a test that ensures it runs.  This is the absolute bare minimum required to push this ticket to a later milestone.  Substantive tests (maybe just one?) would be nice for the next release.

**Comment by icecube-trac** [comment]
Trac update at 20160215T13:10:21: 
* kjmeagher changed resolution from "" to "fixed"
* kjmeagher changed status from "assigned" to "closed"
* kjmeagher commented:

> In changeset 141855:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="141855"
> create end to end test for truncated energy
> reads L3 nugen data and compares the result with the original L3 processings
> fixes #1215
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:57: 
* olivas changed milestone from "IceRec V5" to ""
* olivas commented:

> Milestone IceRec V5 deleted
